src/main/scala/cromwell/engine/backend/jes/JesBackendCall.scala

```diff
@@ -112,4 +114,11 @@ class JesBackendCall(val backend: JesBackend,
 
   override def useCachedCall(avoidedTo: BackendCall)(implicit ec: ExecutionContext): Future[ExecutionHandle] =
     backend.useCachedCall(avoidedTo.asInstanceOf[JesBackendCall], this)
+
+  override val pollBackoff = new ExponentialBackOff.Builder()
+    .setInitialIntervalMillis(30.seconds.toMillis.toInt)
+    .setMaxIntervalMillis(2.minutes.toMillis.toInt)
```

Comment:
Perhaps this should return a custom interface rather than `ExponentialBackoff` so we can get the "constant latency and then exponential" behavior we're after.